doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -6,94 +6,131 @@ Cite Test Guide
 A step by step guide to the GeoServer Compliance Interoperability Test Engine (CITE).
 
 .. contents::
-   :depth: 2
+~~~~~~~~~~~~~
 
-Check out CITE tools
---------------------
+Check out CITE suite tools
+--------------------------
 
-The CITE tools are available at https://github.com/jdeolive/geoserver-cite-tools. The README file 
-contains the most update documentation of how to checkout and build the tools. The quick version is::
+.. note:: The CITE tools are available at `Open Geospatial Consortium`_.
+.. _Open Geospatial Consortium: https://github.com/opengeospatial
+
+Requirements:
+-------------
+
+- `GeoServer <https://github.com/geoserver/geoserver>`_.
+
+- :ref:`Teamengine Web Application<Teamengine Web Application>`
+
+.. note::
+
+   To build every suite test you need to do the next steps:
+
+  .. code:: shell
+
+      git clone https://github.com/opengeospatial/ets-wcs10.git
```

Comment:
I thought these were already built by the Jenkins server.
If they are, I'd update the guide to say so, while still show how to build them in case one need to upgrade.

The index is the first file that is seen by someone walking here, can we have a bit of overview of how things are organized before getting into how to test a specific suite? Maybe it's in the guide, but I see it's at the very end.